gobblin-runtime/src/main/java/gobblin/runtime/util/JobStateToJsonConverter.java

```diff
@@ -238,4 +265,11 @@ public static void main(String[] args)
       System.out.println(stringWriter.toString());
     }
   }
+
+  private static Properties fileToProperties(String fileName) throws IOException, ConfigurationException {
```

Comment:
Can you move this method to `JobConfigurationUtils.java` it seems to be the same as `CliOptions.fileToProperties`.